less/index.less

```diff
@@ -47,13 +49,208 @@ table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
+html {
+	font-size: 62.5%;
+}
 
 /* Set every element's box-sizing to border-box */
 * {
     box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+
+}
+
+@italics: italic;
+@headers: 2rem;
+@paragraphfont:1.9rem;
+@headercolor: pink;
+
+.container {
+	// border: 1px solid rgb(40, 176, 197);
+	max-width: 100%px;
+	margin: 0 auto; 
+	display: flex;
+	flex-wrap: wrap; 
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+header {
+	// border: 1px solid blue;
+	width: 100%;
+	display: flex;
+	flex-wrap: wrap;
+}
+
+header nav {
+	// border: 1px solid green;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-around;
+	margin-top: 30px;
+	width: 100%;
+}
+
+header nav a {
+	// border: 1px solid pink;
+	color: pink;
+	text-decoration: none;
+	font-size: 2rem;
+}
+
+.welcome {
+	// border: 1px dotted orange;
+	width:60%;	
+	margin-bottom: 20px;
+	border-bottom: 2px solid black;
+	h1 {
+		font-size: @headers;
+		// border: 1px solid red;
+		font-style: @italics;
+		display: flex;
+		justify-content: center;
+		font-size: 4rem;
+		flex-wrap: wrap;
+		align-items: center;
+	}
+	#image-one {
+		// width: 370px;
+		height: 235px;
+		margin-top: 10px;
+		margin-bottom: 10px;
+		display: flex;
+	}
+	#image-one img{
+		max-width: 100%;
+		max-height: 100%;
+		margin: auto;
+		display: flex;
+		border-radius: 45%;
+		// border: 10px solid transparent;
+	}
+}
+
+.mainContent {
+	display: flex;
+	flex-direction: column;
+	// border: 1px dotted olivedrab;
+	width:60%;	
+}
+
+
```

Comment:
I noticed that your recycling a lot of the code, below this point, for your h1's. This is a perfect opportunity to use a mixin! 

Let's create a mixin above your aboutMe class with the code from your first h1. Try this -
```
.h1Settings(){
	color: @headercolor;
		font-size: @headers;
		font-style: @italics;
		width: 100%;
		display: flex;
		justify-content: center;
		flex-wrap: wrap;
		align-items: center;              //---------------------------------------------
}
```
Now that we've set our mixin, let's see how many times we can use it.